public/static/docs/command-reference/get.md

```diff
@@ -181,3 +188,16 @@ Untracked files:
 	model.bigrams.pkl
 	model.monograms.pkl
 ```
+
+### Example: Getting a url for a file in a DVC repository
+> Note that `dvc get` can be used from anywhere in the file system, as long as
```

Comment:
have some doubts ... does not feel we need to repeat it here ... why did you decide to put it?

Reply:
You mean the note about "can be used from anywhere in the file system" right Ivan? Not the whole example

Reply:
Yaah, agree about the note. No need 🙂

Reply:
Removed in 072defa3.